android/scripts/make_rust.sh

```diff
@@ -1,52 +1,75 @@
-#!/bin/bash
+#!/bin/bash 
+
+# to debug the script add " -x" to the shebang. Ie. "#!/bin/bash -x"
+
+# get script folder
+SCRIPT_FOLDER=$( cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P )
```

Comment:
Nice! This was basically the goal of https://github.com/Co-Epi/app-backend-rust/issues/103 😉